client/tailscale/localclient.go

```diff
@@ -946,6 +946,21 @@ func (lc *LocalClient) NetworkLockForceLocalDisable(ctx context.Context) error {
 	return nil
 }
 
+// NetworkLockVerifySigningDeeplink verifies the network lock deeplink contained
+// in url and returns information extracted from it.
+func (lc *LocalClient) NetworkLockVerifySigningDeeplink(ctx context.Context, url string) (*tka.DeeplinkValidationResult, error) {
```

Comment:
can you move the verb to the front like the http path on line 956?

VerifyNetworkLockSigningDeeplink?

Reply:
oh, all the other ones start with "NetworkLock" 😢 

nevermind my comment I guess.